type-definitions/most.d.ts

```diff
@@ -242,8 +242,15 @@ export function continueWith<A>(f: (a: any) => Stream<A>, s: Stream<A>): Stream<
 export function concatMap<A, B>(f: (a: A) => Stream<B>, s: Stream<A>): Stream<B>;
 export function mergeConcurrently<A>(concurrency: number, s: Stream<Stream<A>>): Stream<A>;
 
-export function merge<A>(...ss: Array<Stream<A>>): Stream<A>;
-export function mergeArray<A>(streams: Array<Stream<A>>): Stream<A>;
+export function merge<A, B>(...arr: Array<A | B>): Stream<A | B>;
```

Comment:
I have not seen any TypeScript projects that prefix everything with the `I` character. I don't find collisions too much an issue since you can do `import { X as Y } from './X'` 

@davidchase @britancavalier Any thoughts ^^